sklearn/compose/_column_transformer.py

```diff
@@ -372,6 +376,53 @@ def _fit_transform(self, X, y, func, fitted=False):
             else:
                 raise
 
+    def _calculate_inverse_indices(self, X):
+        """
+        Private function to calcuate indicies for inverse_transform
+        """
+        # checks for overlap
+        all_indexes = set()
+        input_indices = []
+        for name, trans, cols in self.transformers:
+            col_indices = _get_column_indices(X, cols)
+            col_indices_set = set(col_indices)
+            if not all_indexes.isdisjoint(col_indices_set):
+                self._invert_error = ("Unable to invert: transformers "
+                                      "contain overlaping columns")
```

Comment:
overlaping -> overlapping

Reply:
I also think the "Unable to invert" is already included into the message in `inverse_transform` ?